examples/autoscaler-webhook/go.mod

```diff
@@ -47,4 +48,4 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
-replace agones.dev/agones => ../../
\ No newline at end of file
+replace agones.dev/agones => ../../
```

Comment:
@markmandel Here's a replace that should answer your question.

Reply:
replace will try to invalidate the versioning constraints of require, and the contents of replace will always prevail.